packages/api-client/src/pdrs.ts

```diff
@@ -2,6 +2,34 @@ import { ApiPdr } from '@cumulus/types/api/pdrs';
 import { invokeApi } from './cumulusApiClient';
 import { ApiGatewayLambdaHttpProxyResponse, InvokeApiFunction } from './types';
 
+/**
+ * Delete a PDR from the Cumulus API
+ *
+ * @param {Object} params            - params
+ * @param {string} params.prefix     - the prefix configured for the stack
+ * @param {string} params.pdrName    - a PDR name
+ * @param {Function} params.callback - async function to invoke the api lambda
+ *                                     that takes a prefix / user payload.  Defaults
+ *                                     to cumulusApiClient.invokeApi
+ * @returns {Promise<Object>}        - the execution fetched by the API
+ */
+export const deletePdr = async (params: {
```

Comment:
This is needed to clean up a collection's dependencies, and.... we're going to need it for users eventually.